sched/scheduler/task_runner_test.go

```diff
@@ -58,7 +99,7 @@ func Test_runTaskAndLog_FailedToLogStartTask(t *testing.T) {
 	sagaCoord := saga.MakeSagaCoordinator(sagaLogMock)
 	s, _ := sagaCoord.MakeSaga("job1", nil)
 
-	err := runTaskAndLog(s, workers.MakeSimWorker(tmp), "task1", task, false, stats.NilStatsReceiver())
+	err := testTaskRunner(s, workers.MakeDoneWorker(tmp), "task1", task, false).run()
```

Comment:
Yup; with simworker everything fails (because the generated jobs have nonsense commands)